deploy/scripts/src/testIPVCore.ts

```diff
@@ -486,11 +541,15 @@ export function coreScenario1 (): void {
       )
       const endTime1 = Date.now()
 
-      check(res, {
+      let passed = check(res, {
```

Comment:
```suggestion
      passed = check(res, {
```
This needs to consistent with the others